src/graph/util/ExpressionUtils.cpp

```diff
@@ -1205,7 +1205,40 @@ bool ExpressionUtils::checkExprDepth(const Expression *expr) {
     }
   }
   return true;
-}  // namespace graph
+}
+
+/*static*/ bool ExpressionUtils::isVidPredication(const Expression *expr) {
+  if (DCHECK_NOTNULL(expr)->kind() != Expression::Kind::kRelIn &&
+      expr->kind() != Expression::Kind::kRelEQ) {
+    DLOG(ERROR) << "DEBUG POINT:0";
+    return false;
+  }
+  const auto *relExpr = static_cast<const RelationalExpression *>(expr);
+  if (relExpr->left()->kind() != Expression::Kind::kFunctionCall) {
+    DLOG(ERROR) << "DEBUG POINT:1";
+    return false;
+  }
+  const auto *fCallExpr = static_cast<const FunctionCallExpression *>(relExpr->left());
+  if (fCallExpr->name() != "id" || fCallExpr->args()->numArgs() != 1 ||
+      fCallExpr->args()->args().front()->kind() != Expression::Kind::kLabel) {
+    DLOG(ERROR) << "DEBUG POINT:2";
+    return false;
+  }
+  if (expr->kind() == Expression::Kind::kRelIn) {
+    // id(V) IN [List]
+    if (relExpr->right()->kind() != Expression::Kind::kList) {
+      DLOG(ERROR) << "DEBUG POINT:3";
+      return false;
+    }
+  } else if (expr->kind() == Expression::Kind::kRelEQ) {
+    // id(V) = Value
+    if (relExpr->right()->kind() != Expression::Kind::kConstant) {
+      DLOG(ERROR) << "DEBUG POINT:4";
```

Comment:
We don't need these debug log if we had done debug.

Reply:
Fixed.